runbotics/runbotics-scheduler/src/scheduler-database/tenant/tenant-invite-code.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class TenantInviteCode {
+    @PrimaryGeneratedColumn('uuid', { name: 'invite_id' })
+    inviteId: string;
+
+    @Column({ type: 'uuid', name: 'tenant_id' })
+    tenantId: string;
+
+    @Column({ type: 'timestamp without time zone', name: 'creation_date' })
```

Comment:
use only timestamp with time zone, unless it's just mapping of the existing entity